ProjFS.Mac/PrjFSKext/KauthHandler.cpp

```diff
@@ -603,7 +603,7 @@ KEXT_STATIC int HandleFileOpOperation(
                         currentVnode,
                         vnodeFsidInode,
                         newPath,
-                        (messageFromProvider && targetRoot == fromRoot) ? fromPath : nullptr, // Only send "from" path if in the same root.
+                        (messageFromProvider && targetRoot == fromRoot) ? fromPath : "", // Send "", to specify there is a fromPath but it is not relavent
```

Comment:
Nit:  Suggestion on rewording the comment:

`// Send "" to specify that the fromPath is not in the same root`